MapboxNavigation/RouteMapViewController.swift

```diff
@@ -287,6 +288,29 @@ class RouteMapViewController: UIViewController {
             mapView.removeArrow()
         }
     }
+
+    func updateCameraAltitude(for routeProgress: RouteProgress) {
+        guard mapView.tracksUserCourse else { return } //only adjust when we are actively tracking user course
+        
+        let zoomOutAltitude = NavigationMapView.zoomedOutMotorwayAltitude
+        let defaultAltitude = NavigationMapView.defaultAltitude
+        let isLongRoad = routeProgress.distanceRemaining >= NavigationMapView.longManeuverDistance
+        
+        
+        let currentStepIsMotorway = currentStep.isMotorway
+        let nextStepIsMotorway = upComingStep?.isMotorway ?? false
+        
+        if currentStepIsMotorway, nextStepIsMotorway, isLongRoad {
```

Comment:
What if the user starts on a motorway and the next step is not a motorway? Shouldn't we zoom out?

Reply:
The most likely scenario where this would happen is if a user _enters_ on one onramp and immediately _exits_ via the next offramp (never actually getting on the interstate), the camera would not zoom out in this instance, which is arguably desirable.

Reply:
(I've had this happen to me a couple of times in denver, not on voyage)